layouts/partials/tw/footer.html

```diff
@@ -17,26 +17,36 @@
           {{ partial "helper/svg.html" "banner" }}
         </svg>
       </a>
+
       <a
         data-sf-campaign="701Dn000000YgogIAC"
         data-ga-category="donate"
         class="
-            inline-flex items-center whitespace-nowrap rounded-md bg-red p-2 text-xs font-semibold uppercase leading-none tracking-wide text-white duration-300 hover:bg-blue sm:flex sm:w-44 sm:justify-center sm:p-3
-            sm:text-lg md:w-64 md:p-4 md:text-xl
+           group relative inline-flex items-center justify-around overflow-hidden rounded-md bg-gradient-to-r from-red-7 to-red p-2 hover:from-blue-7 hover:to-blue 
+           sm:flex sm:w-32 sm:justify-center md:w-52 md:px-7 md:py-4
           "
         href="/donate"
       >
-        <svg
-          class="mr-2 h-4 w-4 fill-current sm:h-6 sm:w-6 md:mr-6 md:h-9 md:w-9"
-          aria-hidden="true"
+        <span
+          class="
+          absolute inset-0 z-0 bg-red [clip-path:_polygon(0_0,_0%_100%,_15%_50%)] group-hover:bg-blue
```

Comment:
`[clip-path:_polygon(0_0,_0%_100%,_15%_50%)]` seems like it might be useful enough to add to tw.css as `.clip-circle` something.

Reply:
Resolved: https://github.com/spotlightpa/poor-richard/pull/422/commits/c37fc3caf4d4fa73ea3a4f5771c85759067bdee8